sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -789,34 +792,38 @@ def _to_certificate_policy_bundle(self):
 
     @classmethod
     def _from_certificate_policy_bundle(cls, certificate_policy_bundle):
-        # type: (models.CertificatePolicy) -> CertificatePolicy
+        # type: (Optional[models.CertificatePolicy]) -> CertificatePolicy
         """Construct a CertificatePolicy from an autorest-generated CertificatePolicy"""
-        if certificate_policy_bundle.lifetime_actions:
+        if certificate_policy_bundle and certificate_policy_bundle.lifetime_actions:
```

Comment:
Instead of checking if the `certificate_policy_bundle` is none throughout the method, it would probably be easier to, at the start of the method, return a `cls()` if `certificate_policy_bundle is None` since all fields will be None by default. Then the rest of the method logic doesn't need to be changed, apart from maybe a mypy comment if there's still a warning.